src/NUnitFramework/tests/nunit.framework.tests.csproj

```diff
@@ -65,4 +74,6 @@
     <ProjectCapability Include="TestContainer" />
   </ItemGroup>
 
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" Condition="$(UseWindowsFormsAndWPF) == false" />
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk.WindowsDesktop" Condition="$(UseWindowsFormsAndWPF)" />
```

Comment:
I can't find it, but I think I read a while back that MSBuild boolean property comparisons should only ever look like one of these. This is also the only style I've seen in the SDK implementation props and targets:

`'$(UseWindowsFormsAndWPF)' == 'true'`
`'$(UseWindowsFormsAndWPF)' != 'true'`

You get an error if you don't put quotes around the property and it evaluates as empty. It can't be empty at this moment, but that's not an obvious constraint for people who may rearrange the csproj later.

(5 occurrences total)

This would also permit us to get rid of the line `<UseWindowsFormsAndWPF>false</UseWindowsFormsAndWPF>`.

Reply:
That's totally right. Will change it.

Reply:
I hope so. It would be nice to find where I read that.